tools/_release.sh

```diff
@@ -30,24 +30,20 @@ echo Releasing production version "$version"...
 nextversion="$2"
 RELEASE_BRANCH="candidate-$version"
 
-if [ "$RELEASE_OPENSSL_PUBKEY" = "" ] ; then
-    RELEASE_OPENSSL_PUBKEY="`realpath \`dirname $0\``/eff-pubkey.pem"
```

Comment:
I think we can delete `tools/eff-pubkey.pem` now too. I think this can be done in a different PR if you prefer that.

Reply:
Nice catch!